manual_block_election.py

```diff
@@ -33,9 +38,8 @@ def main():
         ctx = testctx
 
     peers = filter(lambda p: p.is_voting, get_peers_from_service(ctx))
-    genesis_block = block_open(ctx['genesis_block']['source'], ctx['genesis_block']['representative'],
-                               ctx['genesis_block']['account'], ctx['genesis_block']['signature'],
-                               ctx['genesis_block']['work'])
+    block_hash, root = args.hash.split(':')
```

Comment:
Allow the second part of the root to be optional, have a look at confirm_req.py does it